index.html

```diff
@@ -2162,20 +2207,21 @@ <h6 id="ActionStatus"><code>ActionStatus</code> object</h6>
           </table>
 
           <h6 id="sync-action-response">Synchronous Action Response</h6>
-          <p class="rfc2119-assertion" 
+	  <span class="rfc2119-assertion" 
             id="http-baseline-profile-protocol-binding-invokeaction-13">
+		  <p>
             If providing a Synchronous Action Response, a Web Thing MUST send 
             an HTTP response with:
-          </p>
-          <ul class="rfc2119-assertion" 
-            id="http-baseline-profile-protocol-binding-invokeaction-14">
```

Comment:
`<ul>` tag is now missing.

Reply:
Ugh, this entire table is a combination of saying that things are optional or mandatory AND a set of constraints on their values, which is equivalent to multiple assertions.  I may have to restructure this completely and pull the constraint assertions out of the table.  Also now I can't assume that a table means that something is mandatory....

Reply:
although my comments about the table refer to invokeaction-12, where 2170 is now, so it looks like my previous edits have made line numbers unreliable.  Anyway, invokeaction-14 no longer exists, so... I assume you are referring to invokeaction-13, where I had a </ul> where there should have been a <ul>